src/main/java/core/basesyntax/ColorSupplier.java

```diff
@@ -1,7 +1,10 @@
 package core.basesyntax;
 
+import java.util.Random;
+
 public class ColorSupplier {
-    public String getRandomColor() {
-        return null;
+    public static Colors getRandomColor() {
+        Random random = new Random();
```

Comment:
It is a good practice to reuse objects, like `Random`, consider creating a private final instance variable for `Random` and use it in the `getRandomColor` method. This will improve the performance of the method when it's called multiple times.